server/src/test/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -658,9 +656,10 @@ public void testSnapshotRecovery() throws Exception {
 
         ensureGreen();
 
-        var snapshotInfo = client()
-            .execute(GetSnapshotsAction.INSTANCE, new GetSnapshotsRequest(REPO_NAME, new String[]{SNAP_NAME}))
-            .get().getSnapshots().get(0);
+        execute("SELECT id from sys.snapshots WHERE repository = ? AND name = ?", new Object[]{REPO_NAME, SNAP_NAME});
+
+        String uuid = (String) response.rows()[0][0];
+        SnapshotId snapshotId = new SnapshotId(SNAP_NAME, uuid);
 
```

Comment:
Not using new method `snapshotInfo` as we need only id field here.

If using a common method is more important than selecting required fields only, I  can use it. 
Then will move it from `AbstractSnapshotIntegTestCase` to `IntegTestCase` to make it visible in `IndexRecoveryIT`

Reply:
fine as is